### DISCUSSION TITLE
Is there any new development on flexdashboard for quarto?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#5727] [Q&A] [answered]
### Description

Is there any new development on flexdashboard for quarto?

**Comment by mcanouil** [comment, marked as answer]
Closing this as duplicate of the previous time your asked #2562 and the one before that #843.

When something will be ready, Quarto team will announce it.

Thank you for your interest in Quarto.

**Comment by ManuelSpinola** [comment]
Thank you very much.

El lun, 29 may 2023 a las 7:39, Mickaël Canouil ***@***.***>)
escribió:

> Closing this as duplicate of the previous time your asked #2562
> <https://github.com/quarto-dev/quarto-cli/discussions/2562> and the one
> before that #843
> <https://github.com/quarto-dev/quarto-cli/discussions/843>.
>
> When something will be ready, Quarto team will announce it.
>
> Thank you for your interest in Quarto.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/5727#discussioncomment-6029477>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBZRDHZSUISDQA7KA5DXISRKNANCNFSM6AAAAAAYSZ5QTY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app
